server/events/vcs/azuredevops_client.go

```diff
@@ -130,11 +130,11 @@ func (g *AzureDevopsClient) CreateComment(repo models.Repo, pullNum int, comment
 	return nil
 }
 
-func (g *AzureDevopsClient) ReactToComment(repo models.Repo, pullNum int, commentID int64, reaction string) error { //nolint: revive
```

Comment:
The nolint might be better here since this function is a stub. Once this function is filled out, then the function arguments should be consistent with the other functions.

Reply:
Ok, I'll ignore all the lint errors on this file.